mofacts/server/methods.js

```diff
@@ -2139,6 +2134,34 @@ Meteor.startup(async function() {
       return allErrors;
     },
 
+    //handle file deletions
+    deleteStimFile: async function(stimFilename) {
+      console.log('delete Stim File', stimFilename);
+      const query = 'DELETE FROM item WHERE stimulusFilename = $1';
+      await db.none(query, [stimFilename]);
+      const query2 = 'SELECT stimuliSetId FROM item WHERE stimulusFilename = $1 LIMIT 1';
+      await db.none(query2, [stimFilename]);
+      res = "Stim deleted.";
+      return res;
```

Comment:
We can have stim files without tdfs but we cannot have tdfs without stim files. When a user deletes a stim is should also delete all related tdfs